Trappist/src/Promact.Trappist.Core/Controllers/AccountController.cs

```diff
@@ -1,17 +1,40 @@
-﻿using Microsoft.AspNetCore.Mvc;
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Promact.Trappist.DomainModel.ApplicationClasses.Account;
+using Promact.Trappist.Web.Models;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace Promact.Trappist.Core.Controllers
 {
+    [Authorize]
     public class AccountController : Controller
-    {
-        public IActionResult Login()
-        {
-            return View();
-        }
+    { 
+        // GET: /Account/Login
```

Comment:
No commented lines should be there in a file. Please remove them, If you have any reason to put it here then explain it to me.